src/main/java/com/linkedin/kmf/services/configs/TopicManagementServiceConfig.java

```diff
@@ -76,7 +77,7 @@
               ConfigDef.Importance.LOW, REBALANCE_INTERVAL_MS_DOC)
       .define(PARTITIONS_TO_BROKER_RATIO_CONFIG,
               ConfigDef.Type.DOUBLE,
-              2.0,
+              PARTITIONS_TO_BROKER_RATIO_DEFAULT,
```

Comment:
The two options here are to keep code consistent and have this value stored in two separate places or to break the trend and have a more robust program. I think the latter is a better option